app/commands/v2/put_content.rb

```diff
@@ -40,6 +40,19 @@ def update_content_dependencies(edition)
         update_last_edited_at(edition, payload[:last_edited_at])
         ChangeNote.create_from_edition(payload, edition)
         Action.create_put_content_action(edition, document.locale, event)
+        create_links(edition)
+      end
+
+      def create_links(edition)
+        edition.copy_links_from(previously_published_item.links)
```

Comment:
I think we should only copy links if `payload[:links]` is null. Treat it as a replacement if it is provided or fallback if not provided.